### DISCUSSION TITLE
SUGGESTION : Checkbox for img2img/process batch output filenames to be same as input | Caption checkbox

### DISCUSSION THREAD

**Discussion by yo1nkers** [vladmandic/sdnext#2595] [Ideas]
Hi, I produce a lot of LoRAs and Lyco, and I use this UI's interrogator in the From Image/img2img tab for my LoRA creation.

I like to use the DeepDanBooru interrogator. The DeepDanBooru interrogation within this UI is one of a kind, but it's not available in the script I use to train. I'd love to keep on with this, however I have had to manually interrogate and create a text file for each image. The reason for this is because upon doing a batch interrogation, the filenames of each text file will actually be the interrogation result as well as having the result within the text file. This ends up getting very tedious when I have to manually process up to 30-50 images.

I'd love to see a much faster workaround this by giving us a **quick checkbox which ignores the Image Naming arguments and makes the output filename be the same as the input, even in batch**. 

For example, with the checkbox enabled, if you were batch upscaling 2 images "pic1.png" and "pic2.png", the upscales would go to the set output folder named as "pic1.png" and "pic2.png" **_instead of_** "pic1-upscaled4x.png" and so forth.
As another example, and especially in my use case, I would have 2 images in batch, "pic1.png" and some other pictures, and interrogate them. Rather than them outputting as "1girl, porch, sunset, dense_atmosphere.txt", it would be "pic1.txt"

I would find this incredibly useful, and it would save me a LOT of time. Thanks in advance for reading!

**[TL;DR Section]** What I am asking for is a checkbox in img2img/process that changes the filename output of batch image processing to be the same as the input filenames.

**Comment by yo1nkers** [comment]
It appears it's started randomly titling the names how I want it to be done, but I'd still like to see this added as a feature for more control.

**Comment by vladmandic** [comment]
this is a valid suggestion, converting to feature request so it doesn't fall off the radar.

**Reply by yo1nkers** [reply]
Thank you! Last night a couple more of the batch interrogations ended up unfortunately having the interrogated titles. This feature becoming a reality would indeed be important for me in many cases!
